### PR TITLE
#603: Guest Checkout enabled flag

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #603

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDKTests/ConfigTests.swift [hunk 1]
```diff
@@ -86,4 +86,17 @@ class ConfigTests: XCTestCase {
         expect(config).to(beNil())
     }
 
+    func testGuestCheckoutEnabled() {
+        let options = Options(clientId: clientId, salesChannel: salesChannelId, interfaceLanguage: interfaceLanguage)
```
Comment (bonkey) [inline AtlasSDK/AtlasSDKTests/ConfigTests.swift]: Change around this code is coming: https://github.com/zalando-incubator/atlas-ios/pull/623/commits/0b40d9c961ff5b6009ee3f1bf387e4d26690ad9b#diff-fe420e603acaf0496bd58ac8c074389e

And you might just use Options.forTests()

**Comment by bonkey** [comment]
👍

File: AtlasSDK/AtlasSDKTests/ConfigTests.swift (hunk 1 above)
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDKTests/ConfigTests.swift]: then I need to merge my code first :P

**Pull request merged**